items/PIGS_FOOT.json

```diff
@@ -28,4 +28,4 @@
       "result": "PIGS_FOOT"
     }
   ]
-}
```

Comment:
?

Reply:
I'm not sure why the weird thing happened but I'll fix both issues